From 84bb76a9bc38cc0ece3addcab47936b5c6415c42 Mon Sep 17 00:00:00 2001
From: Brian Wood <brian.j.wood@intel.com>
Date: Thu, 28 Jun 2018 13:46:50 -0700
Subject: [PATCH 1105/1371] Uninitialized return variable reference during
 modeset

During initialization of modeset a ret status variable
could hit a situation where an uninitialized variable
is used to determine if drm mode cleanup is triggered.

Signed-off-by: Brian Wood <brian.j.wood@intel.com>
---
 drivers/gpu/drm/i915/intel_display.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 1ce51f1..d8b62b5 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -14959,7 +14959,7 @@ int intel_modeset_init(struct drm_device *dev)
 	}
 
 	for_each_pipe(dev_priv, pipe) {
-		int ret;
+		int ret = 0;
 
 		if (!i915_modparams.avail_planes_per_pipe) {
 			ret = intel_crtc_init(dev_priv, pipe);
-- 
2.7.4

